web/thesauruses/go/strings.json

```diff
@@ -0,0 +1,287 @@
+{
+  "meta": {
+    "language": "language_id",
+    "language_version": "version.number",
+    "language_name": "Human-Friendly Language Name"
```

Comment:
Can you update these to reflect Go and the version number you're using?

Reply:
Of course. I updated it.